v2/config/samples/compute/v1beta20220301_image.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: compute.azure.com/v1beta20220301
+kind: Image
+metadata:
+  name: aso-sample-image
+  namespace: default
+spec:
+    azurename: asotest-image-hkobbe
+    extendedlocation: null
+    hypervgeneration: V2
+    location: westus2
+    owner:
+        name: asotest-rg-hitcxz
```

Comment:
replace with `aso-sample-rg` to align with other samples

Reply:
Done.